src/cli/cli.cpp

```diff
@@ -1281,6 +1284,35 @@ void Interpreter::ProcessLinkQuality(int argc, char *argv[])
     AppendResult(error);
 }
 
+#if OPENTHREAD_FTD
+void Interpreter::ProcessPSKc(int argc, char *argv[])
+{
+    ThreadError error = kThreadError_None;
+
+    if (argc == 0)
+    {
+        const uint8_t *pskc = otThreadGetPSKc(mInstance);
+
+        for (int i = 0; i < OT_PSKC_MAX_SIZE; i++)
+        {
+            sServer->OutputFormat("%02x", pskc[i]);
+        }
+
+        sServer->OutputFormat("\r\n");
+    }
+    else
+    {
+        uint8_t pskc[OT_MASTER_KEY_SIZE];
```

Comment:
Right now your declaring the variable `pskc` in both the if and the else, while this is valid since each declaration is inside their own scope, could we choose different names for them, or just declare it once at the function scope, to reduce the chance of confusion?